crates/underwriter/src/constraint_submit.rs

```diff
@@ -308,16 +309,18 @@ where
                     let mut i = 0;
 
                     info!("Submitting {txs_len} constraints to relay on  slot {next_slot}");
-                    'submit: while let Err(e) =
-                        relay_client.set_constraints(signed_constraints_message.clone()).await
-                    {
-                        error!(err = ?e, "Error submitting constraints to relay, retrying...");
-                        i += 1;
-                        if i >= max_retries {
-                            error!("Max retries reached while submitting to relay");
-                            break 'submit;
+                    for relay in relay_clients.iter() {
+                        'submit: while let Err(e) =
+                            relay.set_constraints(signed_constraints_message.clone()).await
```

Comment:
We should send all the constraints at the same time, and join for them. Otherwise, it might miss the deadline for some relays.